src/util-device.c

```diff
@@ -271,6 +271,36 @@ const char *LiveGetShortName(const char *dev)
     return live_dev->dev_short;
 }
 
+int CheckAFPacketIPSDevs(void)
+{
+    ConfNode *base = ConfGetNode("af-packet");
+    ConfNode *child;
+
+    if (base == NULL)
+        return 0;
+
+    TAILQ_FOREACH (child, &base->head, next) {
+        ConfNode *subchild;
+        const char *iface = NULL, *copyiface = NULL;
+        TAILQ_FOREACH (subchild, &child->head, next) {
```

Comment:
I don't really understand what we're looping here?

Reply:
From what I understand, each of the entries in a list item like

```
 - interface: wlp0s20f3
    threads: auto 
    cluster-id: 99
    cluster-type: cluster_flow
    defrag: yes
    copy-mode: ips
    copy-iface: eth2   
```